tutorials/circuits/new_getting_started.ipynb

```diff
@@ -0,0 +1,319 @@
+{
```

Comment:
<p>Like the attention brought to the transpilation process. What I'm not completely sure is whether the fake backends are getting-started material. I believe some explanation about them would be needed as well as some list of the available ones (or a link to <a href="https://qiskit.org/documentation/apidoc/providers_fake_provider.html" target="_blank">the API reference</a>). Maybe even some how-to or explanation (in the Diataxis sense) would be nice for fake backends.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1354/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>